en/mkdocs.yml

```diff
@@ -217,6 +217,7 @@ nav:
         - 'Manage Account States':
           - 'Overview': guides/identity-lifecycles/account-states-overview.md
           - 'Lock/Unlock Accounts': guides/identity-lifecycles/lock-account.md
+          - 'Account Locking by Failed Login Attempts': guides/identity-lifecycles/account-locking-by-failed-login-attempts.md
```

Comment:
Shall we make this consistent by having the title as "Lock accounts by failed login attempts" ?